packages/yourloops/test/integration/pages/trends/trends-view-common.spec.tsx

```diff
@@ -104,21 +110,26 @@ describe('Trends view for anyone', () => {
         await checkTimeInRangeStatsTitle()
       })
     })
+
+    it('Should display correct Coefficient of variation stats info with CBG', async () => {
```

Comment:
Is there are reason for doing a new test instead of simply updating: `should render correct tooltips and values` (above)?